.vscode/settings.json

```diff
@@ -5,7 +5,7 @@
     "typescript",
     "typescriptreact"
   ],
-  "eslint.packageManager": "yarn",
+  "eslint.packageManager": "pnpm",
```

Comment:
Can you link me to some docs about what this config applies to? I don't see it in https://code.visualstudio.com/docs/getstarted/settings, so I'm guessing it applies to the eslint extension?

Reply:
Correct, it's here: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint